pyproject.toml

```diff
@@ -46,6 +46,11 @@ keywords = [
 license = {file = "LICENSE"}
 requires-python = ">=3.10"
 
+[project.optional-dependencies]
+slim = ["numba", "pandas"]
```

Comment:
most of this library doesn't depend on the cli, so there _can_ be a slim option. In fact, a better approach would be to move the dependency on `cltoolbox` to an optional dependency tag called `cli = ["cltoolbox"]`.